pkg/handlers/config.go

```diff
@@ -433,6 +514,26 @@ func shouldCreateNewAppVersion(appID string, sequence int64) (bool, error) {
 	return true, nil
 }
 
+func getAppConfigValue(downloadApp *apptypes.App, sequence int64, filename string, archiveDir string) (string, error) {
```

Comment:
how about naming this functions something more specific since it only works for file values, like `getAppConfigValueForFile`?

Reply:
addressed